rllib/evaluation/collectors/agent_collector.py

```diff
@@ -190,16 +190,33 @@ def add_init_obs(
         self._check_view_requirement(SampleBatch.OBS, init_obs)
 
         if SampleBatch.OBS not in self.buffers:
-            self._build_buffers(
-                single_row={
-                    SampleBatch.OBS: init_obs,
-                    SampleBatch.AGENT_INDEX: agent_index,
-                    SampleBatch.ENV_ID: env_id,
-                    SampleBatch.T: t,
-                    SampleBatch.EPS_ID: self.episode_id,
-                    SampleBatch.UNROLL_ID: self.unroll_id,
-                }
-            )
+            single_row = {
+                SampleBatch.OBS: init_obs,
+                SampleBatch.AGENT_INDEX: agent_index,
+                SampleBatch.ENV_ID: env_id,
+                SampleBatch.T: t,
+                SampleBatch.EPS_ID: self.episode_id,
+                SampleBatch.UNROLL_ID: self.unroll_id,
+            }
+
+            # Note (Artur): As long as we have these in our default view requirements,
```

Comment:
TODO to remove when PREV_ACTIONS and PREV_REWARDS get deprecated.

Reply:
Added, thanks!